src/test/java/bio/terra/service/auth/iam/sam/SamIamTest.java

```diff
@@ -397,13 +394,4 @@ public void listAuthorizedResourcesTest() throws Exception {
         samIam.listAuthorizedResources(userReq, IamResourceType.DATASNAPSHOT);
     assertThat(uuidSetMap, is((Map.of(id, Set.of(IamRole.OWNER, IamRole.READER)))));
   }
-
-  @Test
```

Comment:
I see that this test was originally added in the [earlier PR](https://github.com/DataBiosphere/jade-data-repo/pull/1256) for this ticket.  Was the new behavior reverted to the extent that there's no more need for a test?

Reply:
That's right! This PR reverts that change, so this test is no longer accurate. But looking at it again, I realize I could modify it to verify that 401 errors are not retried since we weren't checking that before 🙂